draft-ietf-scitt-architecture.md

```diff
@@ -239,180 +241,182 @@ Reputable Issuers are thus incentivized to carefully review their Statements bef
                    |  Artifact  |
                     '----+-----'
                          v
-                    .----+----.  .----------.
-Issuer       -->   | Statement ||  Envelope  |
-                    '----+----'  '-----+----'
-                         |             |       +------------------+
-                          '----. .----'        | DID Key Manifest |
-                                |              | (decentralized)  |
-                                v              +---+----------+---+
-                             .--+--.   Sign Claim  |          |
-                            | Claim +<------------'           |
-                             '--+--'                          |
-                                |            +--------------+ |
-                             .-' '---------->+ Transparency | |
-                            |   .-------.    |              | |
-Transparency -->            |  | Receipt |<--+   Registry   | |
-     Service                |   '---+---'    +-------+------+ |
-                             '-. .-'                 |        |
-                                |                    |        |
-                                v                    |        |
-                          .-+-------+-.              |        |
-                         | Transparent |             |        |
-                         |    Claim    |             |        |
-                          '-----+-----'              |        |
-                                |                    |        |
-                                |'------.     .------)-------'
-                                |        |   |       |
-                                |        v   v       |
-                                |   .----+---+-----. |
-Verifier      -->               |  / Verify Claim /  |
-                                | '--------------'   |
-                                v                    v
-                       .--------+---------.  .-------+---------.
-Auditor       -->     / Collect Receipts /  / Replay Registry /
-                     '------------------'  '-----------------'
+                    .----+----.  .----------.    DID Identifier
+Issuer       -->   | Statement ||  Envelope  +<------------------.
+                    '----+----'  '-----+----'                     |
+                         |             |           +------------------+
+                          '----. .----'            | DID Key Manifest |
```

Comment:
I suggest maybe not using the words "DID Key Manifest"... 

Maybe "Public Key Identifier" or "VerificationMethod".

Reply:
Captured in #25